bin/gendocs

```diff
@@ -1,5 +1,20 @@
 #!/bin/bash -e
 
+# Automatic document generation, validation and helpers
+#
+# bin/gendocs ...
+# 
+# <unspecified> (default)
+#   Generate HTML documentation from JSON schema. Stored in gencode/docs
+#
+# check
+#   Compares the hashes from the last time gendocs was run 
+#
+# feature_bucket [input_file] [output_file]
+#   Takes a tab delimited text file containing the list of feeature buckets
+#   as copied from Google Sheets and convert into an MD list
```

Comment:
there's nothing fundamentally wrong with this code, but I don't think it's going to be very useful b/c the sheet is *not* going to be the source of truth... over time the MD files will get updated and the sheet itself will quickly get out of date, and then updating using this mechanism will be painful.